airbyte-integrations/connectors/destination-snowflake/src/main/kotlin/io/airbyte/integrations/destination/snowflake/operation/SnowflakeStagingClient.kt

```diff
@@ -18,6 +19,21 @@ private val log = KotlinLogging.logger {}
 /** Client wrapper providing Snowflake Stage related operations. */
 class SnowflakeStagingClient(private val database: JdbcDatabase) {
 
+    private data class CopyIntoTableResult(
+        val file: String,
+        val copyStatus: CopyStatus,
+        val rowsParsed: Int,
+        val rowsLoaded: Int,
+        val errorsSeen: Int,
+        val firstError: String?
+    )
+
+    private enum class CopyStatus {
+        UNKNOWN,
+        LOADED,
+        LOAD_FAILED
```

Comment:
https://docs.snowflake.com/en/sql-reference/sql/copy-into-table#output says there's a "partially loaded" status. currently I think it would be interpreted as UNKNOWN, which we just log and ignore? ... presumably it would be better to throw